dev/js/components/CustomDropdown/index.js

```diff
@@ -0,0 +1,53 @@
+import React, { Component } from 'react';
+import { ddContainer, ddContainerItem, ddContainerItemHeader, optionMenu, headerWrapper, innerWrapper } from './style.js';
+
+
+class CustomDropDown extends Component {
+    state = {
```

Comment:
try to use hooks, or if class based components then try add constructor also, I am not sure about this initialization of state without constructor, if you are too sure about this then it's fine.